src/components/modules/document/common/Upload/modules/Video/modules/Recorder/modules/RecordButton/modules/MediaTimer/MediaTimer.view.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div class="media-timer-view">
+    <div class="media-timer-counter">
+      <svg
+        class="media-timer-svg"
+        :width="`${$options.TIMER_SIZE}px`"
+        :height="`${$options.TIMER_SIZE}px`"
+        :viewbox="`0 0 ${$options.TIMER_SIZE} ${$options.TIMER_SIZE}`"
+      >
+        <path
+          ref="loader"
+          class="media-timer-loader"
+          :transform="
+            `translate(${$options.TIMER_SIZE / 2}, ${$options.TIMER_SIZE / 2})`
+          "
+        />
+      </svg>
+    </div>
+    <span class="media-timer-title">
+      {{ timerTitle }}
+    </span>
+  </div>
+</template>
+
+<script>
+const HALF_PI = Math.PI / 180;
+
+const TIMER_SIZE = 52;
+
+export default {
+  name: 'RecordTimerView',
+
+  TIMER_SIZE,
+
+  props: {
+    countDown: {
+      type: Number,
+      required: true,
+    },
+
+    totalCount: {
```

Comment:
Not clear. I don't understand, what it means

Reply:
renamed, resolved